qa/L0_server_launch_modes/test_config_generator.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2021 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# Copyright (c) 2021-2022 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
@cvinson830 

Regarding whether there is adequate testing for the changes in this PR, I think what we want is a test that confirms that only `c_api` mode validates the `triton_install_path` model analyzer option.

This could be done in a unit test, e.g. `test_entrypoint.py`. A new test could be added that perhaps calls `get_server_handle()` once with each mode, and confirms that on all the non-`c_api` modes, that `validate_triton_install_path()` was _not_ called, and that on the `c_api` mode, that `validate_triton_install_path()` _was_ called. all irrelevant code pieces can be mocked out (including the lower-level filesystem calls)

Thoughts on this unit test idea?

Reply:
Moved to https://jirasw.nvidia.com/browse/TMA-459